src/fare-contracts/details/DetailsContent.tsx

```diff
@@ -60,6 +69,18 @@ export const DetailsContent: React.FC<Props> = ({
     );
 
     const validityStatus = getValidityStatus(now, validFrom, validTo, fc.state);
+
+    const globalMessageRuleVariables = {
+      fareProductType: preassignedFareProduct?.type ?? 'unknown',
+      firstTravelRightType: firstTravelRight.type,
+      validityStatus: validityStatus,
+      tariffZones: firstTravelRight.tariffZoneRefs ?? [],
```

Comment:
When I read the name "tariffZoneRefs", I first thought about react refs. After looking into it, I found that it is a list of ids referencing object id fields in referenceData. Perhaps it should be considered to rename it to tariffZoneRefIds, to both be precise about what it references, and also to avoid confusion with react refs in naming?
I see there are several other variables with the same naming convention, which should also be updated in case we change this..

Reply:
I agree that it's not very intuitive naming, but this naming stems from the firestore (and probably entur) travelrights.

![Screenshot 2023-09-21 at 10 49 53](https://github.com/AtB-AS/mittatb-app/assets/1774972/f2a9ef71-bbf3-49cb-bb79-278f1e1e9f63)

There is probably _some_ code that can be renamed, but I don't think we can get completely rid of it.

Reply:
Following discussion with @gorandalum and @mjenssen, it turns out that the naming convention is defined on our side. However it would require quite a bit of work with versioning for backward compatibility to change, so not worth it at this time.